File: tests/patterns/java/metavar_func_def.java
class Foo {
    // ERROR:
    void foo() {
        foo(1,2);
    }

    // ERROR:
    public static void bar(int bar1, int bar2, int bar3) {
        foo(1,2,3);
        bar(1,2);
    } 

    // ERROR:
    public void foobar(int bar1) {
        foo();
    }

    public static String str() {
        return "hello";
    }
}

File: tests/patterns/csharp/metavar_func_def.cs
public class Test
{
    // ERROR:
    public static int Test()
    {
        Foo(1, 2);
        return 1;
    }

    // ERROR:
    int Test2() {
        return 1;
    }

    // ERROR:
    public static void Main()
    {
        Foo(1, 2);
        return 1;
    }
}
